### PR TITLE
GR-86, add filters[parent_id]

### PR DESCRIPTION
**Pull request by knutsenm**
@twinge

### CODE REVIEW

File: app/models/entity/filter.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ def initialize(entities)
       @entities = entities
     end
 
+    # rubocop:disable MethodLength
```
Comment (knutsenm) [inline app/models/entity/filter.rb]: Help fight discrimination against tall methods!

Comment (twinge) [inline app/models/entity/filter.rb]: :-1:

File: spec/models/entity_filter_spec.rb [hunk 2]
```diff
@@ -69,6 +69,26 @@
...
+      person1 = Entity.create!(person: { client_integration_id: 1 })
+      person2 = Entity.create!(person: { parent_id: person1.id, client_integration_id: 2 })
+      filter1 = Entity::Filter.new(Entity.from('entities as person')
+                              .select('DISTINCT person.*'))
+                              .filter({ 'parent_id' => person1.id }, person_type)
+      expect(filter1).not_to include(person1)
+      expect(filter1).to include(person2)
+    end
+
+    it 'filters on parent_id:not' do
```
Comment (twinge) [inline spec/models/entity_filter_spec.rb]: Is there a use case for this?

Comment (knutsenm) [inline spec/models/entity_filter_spec.rb]: As it happens, with our current data the list of unique ministry parent_ids is quite short, so there is a reasonable use case while doing diagnostics.

**Pull request merged**